.travis.yml

```diff
@@ -15,6 +15,7 @@ install: echo "skip bundle install"
 branches:
   only:
     - master
+    - rewrite_v1
```

Comment:
yes since i am merging all of the rewrite parts into this branch
otherwise travis tests don't run on my PR's :)